### PR TITLE
Rebuild

### PR DESCRIPTION
**Pull request by rickboot**
Rebuilding after another Git disaster.

Still need to re-add data and questions list.

### CODE REVIEW

File: client/src/components/Questions/QuestionsAskBar/QuestionsAskBar.spec.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import QuestionsAskBar from './QuestionsAskBar.jsx';
+import { shallow, mount } from 'enzyme';
+
+describe('QuestionsAskBar unit tests', () => {
```
Comment (ryanminjaeshin) [inline client/src/components/Questions/QuestionsAskBar/QuestionsAskBar.spec.js:6]: Great testing case. Rendering and click function testing. 💯

File: client/src/components/Questions/QuestionsControlBar/Pulldown/Pulldown.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import style from './Pulldown.css';
+
+function Pulldown (props) {
+  return (
+    <label for="sortBy">Sort by:
+      <select name="sortBy" id="sortBy">
```
Comment (ryanminjaeshin) [inline client/src/components/Questions/QuestionsControlBar/Pulldown/Pulldown.jsx:7]: I had to work on this part too haha.

File: client/src/components/Questions/QuestionsControlBar/QuestionsControlBar.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Pulldown from './Pulldown/Pulldown';
+import style from './QuestionsControlBar.css';
+
+function QuestionsControlBar(props) {
+  return (
+    <div className={ style.controlBar }>
+      <div>
+        <span className={ style.label}>Questions</span>
```
Comment (ryanminjaeshin) [inline client/src/components/Questions/QuestionsControlBar/QuestionsControlBar.jsx:9]: I can see one missing space between label & }

File: client/src/components/Questions/QuestionsControlBar/QuestionsControlBar.spec.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import QuestionsControlBar from './QuestionsControlBar.jsx';
+import { shallow, mount } from 'enzyme';
+
+describe('QuestionsControlBar unit tests', () => {
+
+  it ('Renders without crashing', () => {
+    shallow(<QuestionsControlBar />);
+  });
```
Comment (ryanminjaeshin) [inline client/src/components/Questions/QuestionsControlBar/QuestionsControlBar.spec.js:10]: You didn't write the expect statement on purpose here?

File: client/src/components/Questions/QuestionsList/QuestionsList.spec.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import QuestionsList from './QuestionsList.jsx';
+import { shallow, mount } from 'enzyme';
+
+describe('QuestionTopNav unit tests', () => {
+
+  it ('Renders without crashing', () => {
+    shallow(<QuestionsList />);
```
Comment (ryanminjaeshin) [inline client/src/components/Questions/QuestionsList/QuestionsList.spec.js:9]: Also, no expect statement. I am guessing you have not written it on purpose.

File: database/seed.js [hunk 6]
```diff
@@ -27,25 +27,25 @@ const questionSchema = new mongoose.Schema({
...
+//   product_id: String,
+//   user: [questionSchema],
+// })
 
 const genAnswers = (min = 1, max) => {
   const answers = [];
 
-  const rnd = Math.floor(Math.random() * (max - min + 1) + min);
-  for (let n = 0; n < rnd; n += 1) {
+  var rnd = Math.floor(Math.random() * (max - min + 1) + min);
```
Comment (ryanminjaeshin) [inline database/seed.js:38]: I wonder why you change to var from const.

**Review by ryanminjaeshin — APPROVED** [review]
For me, your codes look clean in general. I would love you know if there is reason why you have not written expect statement yet in some testing. :)

File: database/seed.js (hunk 6 above)
Comment (rickboot) [reply database/seed.js:38]: probably part of my git problems. I now have old code in a few places.

**Pull request closed without merging**